lib/Scheduler/sched_region.cpp

```diff
@@ -239,6 +239,9 @@ FUNC_RESULT SchedRegion::FindOptimalSchedule(
     CmputNormCost_(lstSched, CCM_DYNMC, hurstcExecCost, true);
     hurstcCost_ = lstSched->GetCost();
 
+    // Get unweighted spill cost for Heurstic list scheduler
+    HurstcSpillCost_ = lstSched->GetSpillCost();
+
```

Comment:
getSpillCost() has issues when used with SLIL.  It gives you the SLIL cost without subtracting the SLIL bound.  Currently as a consequence of a bug, The lower bound on SLIL is only subtracted from the value returned by GetCost, and is not included in the value returned by GetSpillCost.  (This is not an issue for other cost functions because no other cost functions have no RP cost lower bound).

Reply:
I saw something like this in my other branch, and fixed it there. I didn't see it come up in testing for this PR though -- I'll double check and apply fix if needed.

Reply:
We have actually elected to use weighted cost function for SLIL in two-pass for this PR, this is reflected in my most recent push.